engine/src/engine.rs

```diff
@@ -975,6 +1117,24 @@ pub fn compute_block_hash(chain_id: [u8; 32], block_height: u64, account_id: &[u
     sdk::sha256(&data)
 }
 
+pub fn get_native_erc20_state<I: IO>(io: &I) -> Result<NativeErc20EngineState, EngineStateError> {
+    match io.read_storage(&bytes_to_key(
+        KeyPrefix::Config,
+        NATIVE_ERC20_BRIDGE_STATE_KEY,
+    )) {
+        None => Err(EngineStateError::NotFound),
+        Some(bytes) => NativeErc20EngineState::try_from_slice(&bytes.to_vec())
+            .map_err(|_| EngineStateError::DeserializationFailed),
+    }
+}
+
+pub fn set_native_erc20_state<I: IO>(io: &mut I, state: NativeErc20EngineState) {
+    io.write_storage(
+        &bytes_to_key(KeyPrefix::Config, NATIVE_ERC20_BRIDGE_STATE_KEY),
+        &state.try_to_vec().expect("ERR_SER"),
```

Comment:
Please, do not `panic`. Return `Result` type and more explicit error message.

Reply:
@mrLSD  Should I also fix all panics in this file like this https://github.com/aurora-is-near/aurora-engine/blob/develop/engine/src/engine.rs#L998? 
Because it is weird that panic is not recommended in the engine but part of the code is already used panic.

Reply:
I think, sure.

Reply:
I think we should consider what could cause the error to happen and whether or not it is "fatal". In the case of "fatal" errors it is ok to panic because it is better to crash quickly instead of continue running in an inconsistent state. On the other hand, normal errors should be passed via the return value because they could potentially be handled upstream by the caller.

In this particular example, I think the panic is ok because I think the only way for `try_to_vec` to fail is if it could not allocate enough memory to serialize the object. If the Engine is running out of memory it is probably best to simply shut it down.